packages/functional-tests/tests/signin/signinCached.spec.ts

```diff
@@ -66,29 +66,32 @@ test.describe('severity-2 #smoke', () => {
 
     test('expired cached credentials', async ({
       target,
-      syncBrowserPages: { page, configPage, settings, signin },
+      syncBrowserPages: { page, settings, signin },
       testAccountTracker,
     }) => {
+      test.skip(
+        true,
```

Comment:
Double checking that true is the right condition. If yes, should this be a fixme since it's not a project or config/feature flag thing?

Reply:
Yes, this is correct and just applied until we switch React signin to full prod rollout (should be just one train).